src/cpu/x64/jit_generator.hpp

```diff
@@ -673,6 +673,16 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
         }
     }
 
+    void uni_vhsubps(const Xbyak::Xmm &x, const Xbyak::Xmm &x2,
```

Comment:
Due to the fact that this is a fork from a third party, I suggest not adding new features here. Each change complicates the rebasing process. You can add these functions to the OpenVINO repo as shown [here ](https://github.com/openvinotoolkit/openvino/blob/fe3382312c041930536c0b6e5b34c485ddd4b464/src/plugins/intel_cpu/src/nodes/kernels/kernel_utils.hpp)

Reply:
There are lots of other PRs that already added additional helper `uni_` routines and they was accepted.
Why exactly this functionality should not be accepted ?

I think to solve issue with this fork, this fork should deliver some part of the new functionality in main oneDNN repository

Reply:
We don't have a goal to update the original oneDNN, but a person who will update our fork will transfer each commit manually in worst case.

Reply:
Guys, for these particular case it is ok to leave new uni_* mnemonics here. We already have bunch of them above stock oneDNN. We will transfer them in one batch to the plugin code later.